src/packets.toit

```diff
@@ -229,17 +229,20 @@ class SubscribePacket extends Packet:
 class SubAckPacket extends Packet implements PacketIDAck:
   static TYPE ::= 9
 
-  packet_id/int
-  qos/int
+  /** The qos value for a failed subscription. */
+  static FAILED_SUBSCRIPTION_QOS ::= 0x80
```

Comment:
So the 127 was just the wrong constant too?

Reply:
The spec says 0x80...